radsat-sk/framework/wrappers/RI2C.c

```diff
@@ -61,7 +61,7 @@ uint32_t i2cBlockingTransmit(uint16_t slaveAddress, const uint8_t* data, uint16_
  * @param size Number of bytes to be read from the I2C slave.
  * @return 0 for success, otherwise review I2C.h for return definition
  */
-uint32_t i2cBlockingRecieve(uint16_t slaveAddress, uint8_t* data, uint16_t size) {
+uint32_t i2cRecieve(uint16_t slaveAddress, uint8_t* data, uint16_t size) {
 
 	uint16_t err = I2C_read(slaveAddress, data, size);
 
```

Comment:
something we should really consider is that: Do we want "0" to be considered a success or would "1" make more sense.

Reply:
I think 0 should be considered a success. Even though many of our libraries use 1 for success, I believe that 0 is considered a success in standard C operations.